content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -1,74 +1,355 @@
+:guilabel:`Fiscal Position` is listed called :guilabel:`Automatic Tax Mapping (AvaTax)`. Click on
+this :guilabel:`Fiscal Position`.
 
-Use the :guilabel:`Avalara Code` field that's available on customers, quotations, and invoices to
-cross-reference data in Odoo and Avatax.
+Ensure that the :guilabel:`Use AvaTax API` checkbox is ticked.
 
-.. seealso::
-   - :doc:`fiscal_positions`
+Optionally, tick the checkbox next to the field labeled: :guilabel:`Detect Automatically`. Should
+this option be ticked, then, Odoo will automatically apply this :guilabel:`Fiscal Position` for
+transactions in Odoo.
+
+Specific parameters, such as :guilabel:`VAT required`, :guilabel:`Foreign Tax ID`,
+:guilabel:`Country Group`, :guilabel:`Country`, :guilabel:`Federal States`, or :guilabel:`Zip Range`
+can be specified below this checkbox.
+
+.. warning::
+   Should the :guilabel:`Detect Automatically` checkbox not be ticked, then each *Customer* will
+   need to have the :guilabel:`Fiscal Position` set on their :guilabel:`Sales and Purchase` tab of
+   the contact record. To do so, navigate to :menuselection:`Sales app --> Order --> Customers` or
+   :menuselection:`Contacts app --> Contacts`. Then select and customer or contact to set the fiscal
+   position on.
+
+   Navigate to the :guilabel:`Sales and Purchase` tab and down to the section labeled
+   :guilabel:`Fiscal Position`. Set the :guilabel:`Fiscal Position` field to the fiscal position
+   for the customer.
+
+   .. seealso::
+      - :doc:`fiscal_positions`
+
+AvaTax accounts
+---------------
+
+Upon selecting the checkbox option for :guilabel:`Use AvaTax API` a new :guilabel:`AvaTax` tab
+appears. Click into this tab to reveal two different settings.
+
+The first setting is the :guilabel:`AvaTax Invoice Account`, while the second is, :guilabel:`AvaTax
+Refund Account`. Ensure both accounts are set for smooth end-of-year record keeping. Consult a
+certified public accountant (CPA) for specific guidance on setting both accounts.
+
+Click :guilabel:`Save` to implement the changes.
+
+Tax mapping
+===========
+
+The AvaTax integration is available on Sale Orders and Invoices with the included AvaTax fiscal
+position.
+
+Product category mapping
+------------------------
+
+Before using the integration, specify an :guilabel:`Avatax Category` on the product categories.
+Navigate to :menuselection:`Inventory --> Configuration --> Product Categories`. Select the product
+category to add the :guilabel:`AvaTax Category` to. Finally next to the field labeled
+:guilabel:`AvaTax Category`, click the drop-down menu and select a category, or :guilabel:`Search
+More...` to find one that isn't connected.
+
+.. image:: avatax/avatax-category.png
+   :align: center
+   :alt: Specify AvaTax Category on products
+
+Product mapping
+---------------
+
+AvaTax Categories may be overridden or set on individual products as well. To set the
```

Comment:
In a note admonition, maybe we can say something like:

If both the product and its category have an AvaTax Category set, the product's AvaTax Category takes precedence.
```suggestion
AvaTax Categories may be set on individual products as well. To set the
```